### ISSUE TITLE
Add typescript definitions

### ISSUE THREAD

**Issue by maxencehenneron** [includable/react-native-map-link#45]
I'm using typescript so adding the definitions might be useful for those who use it as well.

I created them for my project, feel free to add them to the repo:

```
declare module "react-native-map-link" {
    import * as React from 'react';
    import { ViewStyle, StyleProp, ImageStyle, TextStyle } from 'react-native';

    interface Options {
        latitude: number | string
        longitude: number | string
        sourceLatitude?: number
        sourceLongitude?: number
        googleForceLatLon?: boolean 
        googlePlaceId?: number
        title?: string
        app?: string
        dialogTitle: string 
        dialogMessage: string 
        cancelText: string
        appsWhiteList?: string[]
    }

    interface PopupStyleProp {
        container?: StyleProp<ViewStyle>,
        itemContainer?: StyleProp<ViewStyle>,
        image?: StyleProp<ImageStyle>,
        itemText?: StyleProp<TextStyle>,
        headerContainer?: StyleProp<ViewStyle>,
        titleText?: StyleProp<TextStyle>,
        subtitleText?: StyleProp<TextStyle>,
        cancelButtonContainer?: StyleProp<ViewStyle>,
        cancelButtonText?: StyleProp<TextStyle>,
        separatorStyle?: StyleProp<ViewStyle>,
        activityIndicatorContainer?: StyleProp<ViewStyle>
    }

    interface PopupProps {
        isVisible: boolean,
        showHeader?: boolean,
        onCancelPressed: () => void,
        onBackButtonPressed: () => void,
        onAppPressed: () => void,
        style?: PopupStyleProp,
        modalProps?: object,
        options: Options,
        appsWhiteList: string[]
    }

    export function showLocation (options: Options): Promise<void>;
    export class Popup extends React.Component<PopupProps>{}
}
```

**Comment by tschoffelen** [comment]
Great! Any chance you could submit a PR to add these to the repo in the appropriate way? I never use Typescript, so not sure what the conventions are.

Thanks in advance!

**Comment by tschoffelen** [comment]
Closed with PR #46.
